src/applications/vaos/tests/appointment-list/components/AppointmentsPage/TabItem.unit.spec.jsx

```diff
@@ -17,7 +17,6 @@ describe('<TabItem>', () => {
     expect(tree.find('NavLink').props().to).to.equal('upcoming');
```

Comment:
I think you can go ahead and delete this file

Reply:
Done